modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -0,0 +1,42 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id="using-the-assisted-installer_{context}"]
+= Using the Assisted Installer
+
+The {product-title} link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[{ai-full}] is a user-friendly installation solution offered on the link:http://console.redhat.com[Red Hat Hybrid Cloud Console]. The {ai-full} supports the various deployment platforms with a focus on bare metal and vSphere.
+
+The {ai-full} provides installation functionality as a service. This software-as-a-service (SaaS) approach has the following advantages:
+
+* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
+* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
+* *Hosting:* The {ai-full} hosts:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The installer
+* *Streamlined installation workflow:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults and provides the installer as a service, which:
+  - Eliminates the need to install and run the {product-title} installer locally
+  - Ensures the latest version of the installer up to the latest tested z-stream releases. Older versions remain available, if needed
+  - Enables building automation through the API without the need to run the {product-title} installer locally
```

Comment:
ISG says to avoid using "through" as a preposition:
```suggestion
  - Enables building automation by using the API without the need to run the {product-title} installer locally.
```
The second part of this sentence "without the need ..." is already highlighted in another sub-item above this one. Does it need to be restated here, or can it be removed? E.g.
```suggestion
  - Enables building automation by using the API.
```